packages/actions/src/contracts/accounts.ts

```diff
@@ -0,0 +1,28 @@
+import { accountsABI } from '@celo/abis'
+import { Address, getContract, GetContractReturnType, PublicClient, WalletClient } from 'viem'
+
+import { resolveAddress } from './registry'
+
+export type AccountsContract<T extends WalletClient | PublicClient = PublicClient> =
```

Comment:
nit but i like having Type on top

then im thinking we do the function for getting the contract and then and exported methods for interacting with contract